### ISSUE TITLE
Does not work with Microsoft Edge v87 and earlier

### ISSUE THREAD

**Issue by egdegauperaa** [janosh/svelte-multiselect#117]
I noticed while testing something in Browserstack that svelte-multiselect does not work in Microsoft Edge version 87 and older. Is this already known?

![image](https://user-images.githubusercontent.com/36701818/192996320-87f1cca9-d60d-477e-a60b-e010504dc053.png)

**Comment by janosh** [comment]
@egdegauperaa Thanks for reporting. 👍 This was not known. Do you know feature we would have to poly-fill or work around to make it work? If it's not an easy fix, my inclination is to ignore it since MS Edge 87 was released almost 2 years ago on Nov 20, 2020 and hence will have limited usage.

**Comment by egdegauperaa** [comment]
We are using your nice component in a government solution that is publically available. In this scenario we need to support old browsers within reason. We won´t support IE11 since it is deprecated, but anything newer should be supported. I don't know if this is the case for other users of your component, but I'd think it's worth spending a little time looking into this.

**Comment by janosh** [comment]
As I suspected from your screenshot since none of the component's default styles are applied, the problem is that [Edge v87 and below doesn't support the `:where()` selector](https://caniuse.com/mdn-css_selectors_where) which this component uses to [make it's default CSS recessive](https://github.com/sveltejs/svelte/issues/6859), i.e. easily overridable with [global CSS](https://github.com/janosh/svelte-multiselect#with-global-css) without having to use the `!important` CSS flag.

The `:is()` pseudo-class has a similar effect and is supported in [earlier versions of Edge](https://caniuse.com/css-matches-pseudo) but needs to be enabled with the 'Experimental Web Platform features' flag so no real gain from switching to that.

I also didn't find a `:where()` CSS polyfill just now. Didn't look thoroughly so could exist.

But maybe the easiest workaround for now (if you don't use any global CSS to customize this component's UI) is to use [`patch-package`](https://www.npmjs.com/package/patch-package) to apply a regex to `MultiSelect.svelte` that strips all the `:where()` selectors:

```diff
- :where(div.multiselect)
+ div.multiselect
```

**Comment by janosh** [comment]
Note that [v8.0.1](https://github.com/janosh/svelte-multiselect/releases/tag/v8.0.1) reverts this change due to the style override issue reported in #140. If you need support for Edge v87 and earlier, please stick to `7.0.1<=svelte-multiselect<=8.0.0`.

**Comment by Snailedlt** [comment]
Just did a little research, and according to [this](https://gs.statcounter.com/browser-version-market-share/desktop/norway/#monthly-202110-202210) gs.statcounter statistic, the marketshare for Edge versions below v87 is only around 0.01% on desktop

Worldwide it's also only around a 0.2% marketshare on dekstop: [source](https://gs.statcounter.com/browser-version-market-share/desktop/worldwide/#monthly-202110-202210)... If you account for other platforms like phones, tablets and consoles, then it's even less.

So in short. Edge v87 is not super important to support, since it has such a small market-share
